tests/functional/feed_test.php

```diff
@@ -1395,7 +1395,7 @@ protected function assert_feed($params, $data)
 					foreach ($attachments as $i => $attachment)
 					{
 						$content = $crawler->filterXPath("//entry[{$entry_id}]/content")->text();
-						$url = "./download/file.php?id={$attachment['id']}";
+						$url = "http://localhost/download/file.php?id={$attachment['id']}";
```

Comment:
Please take it into account that functional tests could be runned on different paths than `http://localhost` by defining `$phpbb_functional_url`